addons/web_editor/static/src/js/widgets.js

```diff
@@ -482,20 +486,25 @@ var ImageDialog = Widget.extend({
     },
     select_existing: function (e) {
         var $img = $(e.currentTarget);
+        if(!this.options.select_images){
+            this.$('.o_existing_attachment_cell.o_selected').removeClass("o_selected");
+            $img.closest('.o_existing_attachment_cell').addClass('o_selected');
+        }
+        else{
+            $img.closest('.o_existing_attachment_cell').toggleClass('o_selected');
+        }
```

Comment:
See javascript guidelines (https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines).
Here use `if (...) {` not `if(...){`
Also same comment for the `else`